DEVELOPER_WALKTHROUGH.MD

```diff
@@ -0,0 +1,53 @@
+# Developer Walkthrough and Project Setup
+
+## Prerequisites
+* Docker
+* Git
+* Java 11+ ([download](https://www.oracle.com/java/technologies/javase-jdk11-downloads.html))
+* Maven ([download](https://maven.apache.org/download.cgi))
+* Powershell ([download](https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell?view=powershell-7.1))
```

Comment:
Now that the startup script is in bash, I don't think Powershell is necessarily a prereq. Perhaps include it as optional?